enterprise_access/apps/subsidy_access_policy/constants.py

```diff
@@ -25,6 +25,16 @@ class SegmentEvents:
     SUBSIDY_ACCESS_POLICY_CREATED = 'edx.server.enterprise-access.enrollment-lifecycle.subsidy-access-policy.created'
     SUBSIDY_ACCESS_POLICY_REDEEMED = 'edx.server.enterprise-access.enrollment-lifecycle.subsidy-access-policy.redeemed'
 
+class PolicyTypes:
```

Comment:
I think we don't need this as I explained earlier. These types are for internal use only.

Reply:
As per new discussion, we do need it.